thirdparty/KDDockWidgets/src/private/DragController.cpp

```diff
@@ -329,6 +329,12 @@ void StateDragging::onEntry()
 void StateDragging::onExit()
 {
     m_maybeCancelDrag.stop();
+
```

Comment:
Previously saveLastFloatingGeometry() was only called when starting the drag. This means the *previous* geometry is saved. So, save it when the drag is finished as well.